### ISSUE TITLE
Alexander

### ISSUE THREAD

**Issue by AlexVOiceover** [fac30/User-Manuals#1]
# Name
Alexander

## Pronouns
He/Him

### How I get my best work done:
I need some time to get my focus on a task. Once I am focused, I have a quite good pace, but switching between tasks is hard for me. Don't worry if, while working on something, you ask me something and I look like I'm on another planet. I just need some time to refocus.

### The role I usually take in a team:
I usually try to fill the gaps. If I see that a team needs some extra effort, I try to fill that role so the team doesn't have a clear weakness. When the team is solid and everything works smoothly, I like to sit on the sidelines and enjoy observing the group dynamics.

### My communication style:
I'm afraid that depends on the day. Sometimes I am very chatty and silly, while other days I find it exhausting just to be around a group of people. I still haven't identified what makes me feel one way or the other.

### What I value:
Respect for yourself, for your team, and for the group/organization you are working in.
I also value curiosity and the ability to have fun with what you're working on.

### What people misunderstand about me:
Working as a sound engineer for the past 14 years rewired my brain in a way that makes it very hard for me to follow a conversation if there is noise around. The noise filtering that most people do automatically works in the exact opposite way for me. That's also why I don't like zoom calls. If I ever give you weird looks, it may be because I cannot decipher your words, and I am fantasizing about being Godzilla and destroying all the trucks, stepping on people yelling on the phone while walking next to you, and kids.

### How I like to get my feedback:
Just be honest. I'm from Spain, we don't know what diplomacy is. The shorter, the better.

### Technical strengths & weaknesses:
My main weakness is my memory. If you ask me for a specific command or the syntax of a particular method, I probably won't be able to remember it.
Regarding strengths, I would say that I find it easy to understand new problems (like new code) and identify where it's failing or what can be improved. I also try to apply methodologies from one knowledge area to fix problems in a completely different area.

### Interests (programming or otherwise!):
Drone building and piloting, 3D printing (from 3D modeling to building and hard modding the 3D printer), sewing, fixing vintage electronics, silly Arduino projects, watchmaking and everything related to sound recording, from fixing or modding mics to building amps.

**Comment by JasonWarrenUK** [comment]
Thought: when we read someone's user manual, we should leave a reaction on it. That way we know people read it.

**Comment by AlexVOiceover** [comment]
Yes, I love the idea. That is something you should remind for your cohort
